### PR TITLE
Fix label styles affecting shabox

### LINKED ISSUE #13704: Commit hash and commiter avatar off in the PR view
- Gitea version (or commit ref): `1.14.0+dev-258-g03fa2eccb`
- Git version: `git version 2.29.2.526.gbb4321173f`
- Operating system: Arch Linux
- Built from
  - [x] source
- Run with
  - [x] systemd
- Database:
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] No

## Description
Recently I have noticed this anomalously off-set commit hash+avatar labels in the PR view (see screenshots below).
Could this be somehow linked to the recent and otherwise awesome improvements brought by https://github.com/go-gitea/gitea/issues/13594?

## Screenshots
image
image

### PR DESCRIPTION
**Pull request by silverwind**
Add common parent .labels-list to issue labels lists to prevent affecting other elements.

Fixes: https://github.com/go-gitea/gitea/issues/13704

Screen Shot 2020-11-28 at 16 14 23
Screen Shot 2020-11-28 at 16 14 31
Screen Shot 2020-11-28 at 16 14 38
Screen Shot 2020-11-28 at 16 14 50

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: templates/shared/issuelist.tmpl [hunk 1]
```diff
@@ -37,9 +37,11 @@
 							{{template "repo/commit_status" (index $.CommitStatus .PullRequest.ID)}}
 						{{end}}
 					{{end}}
-					{{range .Labels}}
-						<a class="ui label" href="{{$.Link}}?q={{$.Keyword}}&type={{$.ViewType}}&state={{$.State}}&labels={{.ID}}{{if ne $.listType "milestone"}}&milestone={{$.MilestoneID}}{{end}}&assignee={{$.AssigneeID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}" title="{{.Description | RenderEmojiPlain}}">{{.Name | RenderEmoji}}</a>
-					{{end}}
+					<div class="labels-list">
```
Comment (silverwind) [inline templates/shared/issuelist.tmpl]:
```suggestion
					<span class="labels-list">
```

File: templates/shared/issuelist.tmpl [hunk 2]
```diff
@@ -37,9 +37,11 @@
...
 						{{end}}
 					{{end}}
-					{{range .Labels}}
-						<a class="ui label" href="{{$.Link}}?q={{$.Keyword}}&type={{$.ViewType}}&state={{$.State}}&labels={{.ID}}{{if ne $.listType "milestone"}}&milestone={{$.MilestoneID}}{{end}}&assignee={{$.AssigneeID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}" title="{{.Description | RenderEmojiPlain}}">{{.Name | RenderEmoji}}</a>
-					{{end}}
+					<span class="labels-list">
+						{{range .Labels}}
+							<a class="ui label" href="{{$.Link}}?q={{$.Keyword}}&type={{$.ViewType}}&state={{$.State}}&labels={{.ID}}{{if ne $.listType "milestone"}}&milestone={{$.MilestoneID}}{{end}}&assignee={{$.AssigneeID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}" title="{{.Description | RenderEmojiPlain}}">{{.Name | RenderEmoji}}</a>
+						{{end}}
+					</div>
```
Comment (silverwind) [inline templates/shared/issuelist.tmpl]:
```suggestion
					</span>
```

**Review by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf — APPROVED** [review]
applied the patch.
fixes the issue (#13704) on my instance 🎉 
thanks @zeripath

**Review by lunny — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**